moveit_ros/perception/pointcloud_octomap_updater/src/pointcloud_octomap_updater.cpp

```diff
@@ -46,6 +46,9 @@
 
 #include <memory>
 
+using boost::placeholders::_1;
```

Comment:
I would prefer having these specified in the bind commands explicitly. Eventually, we will switch to `std::bind()` or lambdas anyway.